src/Catel.Test/Catel.Test.Shared/MVVM/Commands/TaskCommandFacts.cs

```diff
@@ -35,11 +35,19 @@ public class TaskCommandFacts
             taskCommand.Execute();
 
             Assert.IsTrue(taskCommand.IsExecuting);
+#if NET40
+            await TaskEx.Delay(TimeSpan.FromSeconds(1));
+#else
             await Task.Delay(TimeSpan.FromSeconds(1));
```

Comment:
Not sure, but I think you can use ThreadHelper.Sleep for this. It works for all platforms (saves you a lot of #if stuff)